packages/devkit/src/utils/async-iterable/combine-async-iterables.spec.ts

```diff
@@ -143,4 +145,40 @@ describe('combineAsyncIterables', () => {
       expect(e.message).toMatch(/threw in b/);
     }
   });
+
+  it('should handle throws from runExecutor which uses combineAsyncIterable and eachValueFrom', async () => {
+    // ARRANGE
+    const subjectThatWontThrow = new Subject<string>();
+    const iter = runExecutor(subjectThatWontThrow);
+    const subjectThatThrows = new Subject<string>();
+    const iter2 = runExecutor(subjectThatThrows);
+
+    // ACT
+    const c = combineAsyncIterables(iter, iter2);
+    subjectThatWontThrow.next('from test');
+    subjectThatThrows.next('from test');
+    subjectThatThrows.error('my error');
+
+    // ASSERT
+    try {
+      for await (const _x of c) {
+        // do nothing
+      }
+    } catch (e) {
+      expect(e).toMatch(/my error/);
+    }
+  });
 });
+
+async function* runExecutor(subject: Subject<string>) {
```

Comment:
Can we simplify the test to not use rxjs? Also, no need to name the function `runExecutor` since it's a more general util (nit).

Reply:
I can rename the function for sure, but I’m intentionally using RxJS to closely match the behaviour that was erroring before which was using RxJS w/ eachValueFrom.

I can potentially try to replicate it in another way, but I wanted the test to match what we were seeing